indigo_api/models.py

```diff
@@ -1263,8 +1263,8 @@ def may_submit(self, view):
     def submit(self, user):
         if not self.assigned_to:
             self.assign_to(user, user)
-        self.last_assigned_to = self.assigned_to
-        self.assigned_to = None
+        self.submitted_by_user = self.assigned_to
+        self.assigned_to = self.reviewed_by_user
```

Comment:
Shouldn't this be `self.assign_to` so that the assignment is stored as an action and emails are sent?

Reply:
Oh, I see that there's a new email for `submit` with a reviewer. I'm not against having  both emails sent: assigned to you, and the task was submitted back to you for review.

Reply:
I'm not crazy about the idea of two emails – I think of it as the converse of 'changes requested', where only one email is sent to the submitted_by_user. Happy to use the assignment instead, the wording of the email was just a bit trickier (I tried that first).

Reply:
Fair point, it mirrors `unsubmit`. Let's do it.